ui/src/views/network/PublicIpResource.vue

```diff
@@ -136,17 +146,22 @@ export default {
         // VPC IPs don't have firewall
         let tabs = this.$route.meta.tabs.filter(tab => tab.name !== 'firewall')
 
+        if (this.resource.networkofferingconservemode) {
+          this.tabs = tabs
+          return
+        }
+
         this.portFWRuleCount = await this.fetchPortFWRule()
         this.loadBalancerRuleCount = await this.fetchLoadBalancerRule()
 
         // VPC IPs with PF only have PF
         if (this.portFWRuleCount > 0) {
-          tabs = this.defaultTabs.concat(this.$route.meta.tabs.filter(tab => tab.name === 'portforwarding'))
+          tabs = tabs.filter(tab => tab.name !== 'loadbalancing')
```

Comment:
Not sure if this is required. Because for conserve mode, we have public IP having both LB and PF rule setup, so it doesn't display both tabs